### ISSUE TITLE
Fix build

### ISSUE THREAD

**Issue by lam2mo** [JMU-CS/less-java#68]
#66 seems to have broken the build. There is some discussion on that pull request, but it's unclear to me what's going on and I'm seeing the failures locally as well. We should fix this ASAP.

**Comment by lam2mo** [comment]
Perhaps this has something to do with Java version? Seems broken on this version of Java:

```
java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)
```

**Comment by lam2mo** [comment]
This seems to be fixed by #70.
